src/Debug/skeletonViewer.ts

```diff
@@ -772,17 +772,18 @@ export class SkeletonViewer {
             let green = new Color4(0, 1, 0, 1);
             let blue = new Color4(0, 0, 1, 1);
             let colors = [[red, red], [green, green], [blue, blue]];
+            let axes = MeshBuilder.CreateLineSystem('localAxes', { lines: lines, colors: colors, updatable: true }, targetScene);
+            
+            axes.parent = b;
+            axes.renderingGroupId = this.renderingGroupId;
 
-            this._localAxes = MeshBuilder.CreateLineSystem('localAxes', { lines: lines, colors: colors, updatable: true }, targetScene);
-
-            if (this.displayMode === SkeletonViewer.DISPLAY_LINES && !this._localAxes.skeleton) {
-                // The local axes needs a mesh. Since the world matrix of a bone is not recalculated unless its
-                // skeleton is applied to at least one mesh. For the other display mode we don't have this issue.
-                this._localAxes.skeleton = this.skeleton;
-            }
+            this._localAxes.push(axes);
+        }
 
-            this._localAxes.parent = b;
-            this._localAxes.renderingGroupId = this.renderingGroupId;
+        if (this.displayMode === SkeletonViewer.DISPLAY_LINES && !this._localAxes[0].skeleton) {
+            // The local axes needs a mesh. Since the world matrix of a bone is not recalculated unless its
+            // skeleton is applied to at least one mesh. For the other display mode we don't have this issue.
+            this._localAxes[0].skeleton = this.skeleton;

```

Comment:
I would have applied the skeleton to all X axis because if `_localAxes[0]` leaves the frustum for whatever reason (animation / user moving the mesh), then the skeleton won't be processed anymore and the axis won't be displayed correctly.

Reply:
I see. Thanks.

Reply:
Question: Would it be good practice to include the pull request ID in each related commit?

Reply:
I don't think so, as nobody does this :)